util/settings/settings.go

```diff
@@ -13,6 +13,7 @@ import (
 	"time"
 
 	"github.com/ghodss/yaml"
+	"github.com/gobwas/glob"
```

Comment:
Golang has `filepath.Match` . It implements `glob` but with a limitation ( * does not match the `\` character ) . We've used it since limitation does not affect us yet. I don't mind switching to better glob implementation. In this case we should stop using `filepath.Match`in https://github.com/argoproj/argo-cd/blob/master/pkg/apis/application/v1alpha1/types.go

Reply:
@jessesuen - thoughts?